src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/GetJsonAttributeValue.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    // Takes a path to a json file and a string that represents a dotted path to an attribute
```

Comment:
This documentation says "dotted" which I interpret as a period yet the implementation is using a colon.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/05164e106ecff2a763c3d52dd13c328dbb35e485